en/user_guide/websites_creation.rst

```diff
@@ -0,0 +1,101 @@
+Website creation
+================
+
+Open Orchestra is multisite and allows to create several websites from the same Back Office application.
+
+Name (required)
+---------------
+
+The name field is used for the website name and it allows to generate the identifier of the website.
+
+Website id (required)
+---------------------
+
+The site id can be generated from the site name or directly by the user at the creation.
+In the remaining process, this field is not anymore editable.
+Open Orchestra verifies during the form validation that the site id does not already exist.
+
+.. image:: ../../images/website_id.png
+
+Aliases
+-------
+
+Open Orchestra allows to create  several aliases for a site.
+
+For example :
+
+* Scheme by default (required): http
+* Domain (required): demo.openorchestra.dev
+* Language (required): English
+* Language prefix : en
+* main aliases : on
+
+The website home page will be ``http://demo.openorchestra.dev/en``.
+
+Open Orchestra uses only the main aliases for the sitemap generation.
+
+.. image:: ../../images/aliases.png
+
+Blocks available
+----------------
+
+This attribute lists all the blocks available for the nodes of this website.
+The selected blocks will be visible during the node contribution.
```

Comment:
Maybe something like "Only the selected blocks will be availables during the node contribution"